common-npm-packages/docker-common-v2/package.json

```diff
@@ -13,13 +13,16 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
+    "@types/node": "10.17.0",
+    "@types/mocha": "5.2.7",
+    "@types/uuid": "8.3.0",
     "del": "2.2.0",
     "q": "1.4.1",
     "vso-node-api": "6.0.1-preview",
```

Comment:
Looks like this package is deprecated in favor of `azure-devops-node-api`. Have you considered to change this?

Reply:
Removed as it is unused.